ts/features/itwallet/common/utils/itwIssuanceUtils.ts

```diff
@@ -340,3 +353,7 @@ const getSpidProductionIdpHint = (spidIdpId: string) => {
   }
   return SPID_IDP_HINTS[spidIdpId];
 };
+
+export const disposeWallet = async () => {
+  await disposeEidCryptoKeys();
+};
```

Comment:
Why is this just a wrapper around `disposeEidCyrptoKeys`? Do you think we might need to add more operations here?

Reply:
We could add more clean up logic in the future, but currently it is just a wrapper as you noted. I removed it to simplify the code in 7d1038c25a288edfd17f382fea8fb73654e7173b.